src/cdomains/intDomain.ml

```diff
@@ -263,12 +306,12 @@ struct
   let shift_right = bit (fun x y -> Int64.shift_right x (Int64.to_int y))
   let shift_left  = bit (fun x y -> Int64.shift_left  x (Int64.to_int y))
 
-  let neg = function None -> None | Some (x,y) -> norm @@ Some (Int64.neg y, Int64.neg x)
+  let neg = function None -> None | Some x -> IntPair.norm @@ Some (IntPair.neg x (0L, 0L))
```

Comment:
?

Reply:
It's again same old thing about making `IntPair.neg` binary; the second argument does actually nothing here.

Reply:
I think if you follow my suggestion on the other one, this would go away, right?